SchoolProject/src/test/java/services/ReadDataServicesTests.java

```diff
@@ -0,0 +1,66 @@
+package services;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import models.Student;
+import models.Teacher;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class ReadDataServicesTests {
+
+    private ReadDataService readDataService;
+
+    @BeforeEach
+    public void initialConfiguration(){
+        this.readDataService = new ReadDataService();
+    }
+
+    @Test
+    public void shouldReadStudentsDataFromJson(){
+        List<Student> students = null;
+
+        try {
```

Comment:
I think it is not bad to use try catch in unit tests but is good to use the assert throws, like we did in class